frontend/src/components/Navigator/ActivityReportNavigator.js

```diff
@@ -405,7 +405,7 @@ const ActivityReportNavigator = ({
       };
 
       if (allowUpdateFormData) {
-        updateFormData(data, true);
+        updateFormData(data, false);
```

Comment:
This fixes a bug where if you "save draft" with a goal filled out (but before you've saved and continued) and then leave the report, the goal will disappear. I believe this is the bug that C'era showed us a video of.